Pod/Classes/MentionComposer.swift

```diff
@@ -262,14 +264,26 @@ public class MentionComposer<T: UIView where T: ComposableAttributedTextContaini
 
         // Check for a mention query
         if let query = mentionQuery(fromString: text) {
+            requestingMentionUsers = true
+            refreshTableView()
+
             delegate?.usersMatchingQuery(query as String) { [weak self] (users) -> Void in
+                guard self?.requestingMentionUsers == true else { return }
+                
+                self?.requestingMentionUsers = false
+
                 if let users = users {
                     self?.userNameMatches = users
-                    self?.refreshTableView()
                 }
+                else {
```

Comment:
I'm used to doing newlines from my objective c days, but we can update the style in alter PR if you want.